binplace.targets

```diff
@@ -10,21 +10,35 @@
     <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
     <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <BinplaceConfiguration Include="$(BuildConfiguration)">
+      <RefPath>$(RefPath)</RefPath>
```

Comment:
I would say do it if you think will not cause any problem. 

---
In reply to: [98114905](https://github.com/dotnet/corefx/pull/15529#discussion_r98114905) [](ancestors = 98114905)